handlebars.js

```diff
@@ -1,22 +1,13 @@
 const Handlebars = require('handlebars');
-const {passing, failed, pending} = require('./constants');
+const {statusMarks, pending} = require('./constants');
 const {formatJSON} = require('./files');
 
 // helpersFile is a path to a project specific helpers file.
 const registerHelpers = (helpersFile) => {
-  Handlebars.registerHelper('getStatusMark', state => {
-    let statusMark = pending;
-    if(state === 'passed') {
-      statusMark = passing;
-    }
-    if(state === 'failed') {
-      statusMark = failed;
-    }
-    return statusMark;
-  });
-  
+  Handlebars.registerHelper('getStatusMark', state => statusMarks[state] || pending);
```

Comment:
```suggestion
  Handlebars.registerHelper(
    'getStatusMark', state => statusMarks[state] || pending);
```

Reply:
should be addressed here: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/7/commits/4585e53dd271bcf777588d558747dd6eb3366102